pyuvm/__init__.py

```diff
@@ -17,8 +17,19 @@
 # Section 14, 15 (Done as fresh Python design)
```

Comment:
I'm not sure what this refers to.

Reply:
sorry, linked to the wrong file. This comment refers to the newly added file names. Current files in repo doesn't has `_uvm_` in the name, whereas the new files have `_pyuvm_`. it would be good to use one naming convention either `_pyuvm_` or `_uvm_`

Reply:
Yeah thinking about how many files do have uvm in the name rather then pyuvm best is to revert back mine to uvm. I’ll do it immediately so I can do a new commit today